{
    "configurations": [
        {
            "name": "Python Debugger: Current File",
            "type": "debugpy",
            "request": "launch",
            "program": "${file}",
            "console": "integratedTerminal"
        },
        {
            "name": "Debug Unit Test",
            "type": "debugpy",
            "request": "launch",
            "justMyCode": false,
            "program": "${file}",
            "purpose": [
                "debug-test"
            ],
            "console": "integratedTerminal",
            "env": {
                // Enable break on exception when debugging tests (see: tests/conftest.py)
                "PYTEST_RAISE": "1",
            },
        }
    ]
}
